### PR TITLE
Formatting changes for Py2-3-compat

### PR DESCRIPTION
**Pull request by MichalMMac**
`black` and friend used on processors.

Also removes several `from __future__ import absolute_import`. Was there a need for them?

### CODE REVIEW

File: AdobeFlashPlayer/AdobeFlashURLProvider.py [hunk 1]
```diff
@@ -129,15 +129,11 @@ def get_adobeflash_dmg_url(self, headers=None, opts=None):
 
     def main(self):
         """Return a download URL for latest Mac Flash Player"""
-
```
Comment (nmcspadden) [inline AdobeFlashPlayer/AdobeFlashURLProvider.py]: I'm not seeing any changes needing to be made by black here:
```
pre-commit run --files AdobeFlashPlayer/AdobeFlashURLProvider.py
black....................................................................Passed
isort....................................................................Passed
seed isort known_third_party.............................................Passed
Flake8...................................................................Passed
```

**Comment by nmcspadden** [comment]
I've merged your other PR, and then updated it from master, so you'll need to rebase this.

**Comment by nmcspadden** [comment]
This should actually no longer be necessary - as of the latest changes, we're up to date and good to go.
```
pre-commit run -av
[black] black............................................................Passed
hookid: black

All done! ✨ 🍰 ✨
15 files left unchanged.

[isort] isort............................................................Passed
[seed-isort-config] seed isort known_third_party.........................Passed
[flake8] Flake8..........................................................Passed
```

**Comment by MichalMMac** [comment]
@nmcspadden You are correct.

**Pull request closed without merging**